root/etc/cont-init.d/50-gid-video

```diff
@@ -3,32 +3,34 @@
 # check for the existence of a video and/or tuner device
 if [ -e /dev/dri ] || [ -e /dev/dvb ]; then
 	if [ -e /dev/dri ]; then
-		VIDEO_GID=$(stat -c '%g' /dev/dri/* | grep -v '^0$' | head -n 1)
-	else
-		VIDEO_GID=$(stat -c '%g' /dev/dvb/* | grep -v '^0$' | head -n 1)
+	FILES="${FILES} /dev/dri/*"
 	fi
-	# just add abc to root if stuff in dri/dvb is root owned
-	if [ -z "${VIDEO_GID}" ]; then
-		usermod -a -G root abc
-		exit 0
+	if [ -e /dev/dvb ]; then
+	FILES="${FILES} /dev/dvb/*"
 	fi
 else
 	exit 0
 fi
 
-# Check if this GID matches the current abc user
-ABCGID=$(getent group abc | awk -F: '{print $3}')
-if [ "${ABCGID}" == "${VIDEO_GID}" ]; then
-	exit 0
-fi
+for i in $FILES
```

Comment:
If this were:
```
for i in "$(find /dev/dri /dev/dvb -type c -print -quit 2>/dev/null)"
```
it would negate the need for this whole chunk https://github.com/linuxserver/docker-plex/blob/177088c2988aa610d0a5b81a3525e08e40d79b36/root/etc/cont-init.d/50-gid-video#L3-L14 and cover the intention of https://github.com/linuxserver/docker-plex/pull/202

Other changes might be appropriate as well since I'm not 100% following the intention of `/groupadd` at a quick glance.

Reply:
Do you mean `find /dev/dri /dev/dvb -type c -print 2>/dev/null` ?

Reply:
Good catch!